cmd/sqlflowserver/main_test.go

```diff
@@ -876,7 +876,7 @@ COLUMN SPARSE(deep_id,15033,COMMA,int),
        EMBEDDING(CATEGORY_ID(user_behavior_stat,511,COMMA),64,mean),
        EMBEDDING(CATEGORY_ID(space_stat,418,COMMA),64,mean)
 LABEL l
-INTO model_table;`, caseDB)
+INTO model_table;`, caseDB, os.Getenv("GITLAB_TOKEN"))
```

Comment:
using `t.Fatalf` if no `GITLAB_TOKEN` in the env to avoid raising the nil point exception.

Reply:
`os.Getenv` will return an empty string if the environment variable is not defined, in that case, the test should fail due to some gitlab error.